bot/exts/pride/drag_queen_name.py

```diff
@@ -5,28 +5,28 @@
 
 from discord.ext import commands
 
+from bot.bot import Bot
+
 log = logging.getLogger(__name__)
 
 
 class DragNames(commands.Cog):
     """Gives a random drag queen name!"""
 
-    def __init__(self, bot: commands.Bot):
-        self.bot = bot
+    def __init__(self):
         self.names = self.load_names()
 
     @staticmethod
     def load_names() -> list:
         """Loads a list of drag queen names."""
-        with open(Path("bot/resources/pride/drag_queen_names.json"), "r", encoding="utf8") as f:
-            return json.load(f)
+        return json.loads(Path("bot/resources/pride/drag_queen_names.json").read_text("utf8"))
```

Comment:
This function doesn't need to exist if it's only used once. Into a constant it goes.